README.md

```diff
@@ -173,16 +119,17 @@ life-science domain data assets.
 
```

Comment:
I dont like the title (I know its not from you)
What is "and more". This also repeats in the next heading and might lead to confusion later on